deploy/lvmd-config/lvmd.yaml

```diff
@@ -3,7 +3,7 @@ device-classes:
   - name: ssd
     volume-group: myvg1
     default: true
-    spare-gb: 10
+    spare-gb: 1
```

Comment:
Please left unchanged here. Instead, please set spare-gb to 1 when copy this file in `example/Makefile`.

Reply:
Fixed.